jib-plugins-common/src/main/java/com/google/cloud/tools/jib/plugins/common/JibRunner.java

```diff
@@ -44,8 +43,8 @@
 import javax.annotation.Nullable;
 import org.apache.http.conn.HttpHostConnectException;
 
-/** Runs a {@link BuildSteps} and builds helpful error messages. */
-public class BuildStepsRunner {
+/** Runs Jib and builds helpful error messages. */
+public class JibRunner {
```

Comment:
Nit: Not sure we need to use `Jib` in the name here. Perhaps `BuildRunner`? I'm okay either way though.

Reply:
The previous commit had this [now-gone comment](https://github.com/GoogleContainerTools/jib/pull/1659/commits/4af8648bfde1c644ece9ccb669748ddb236766b1#diff-ea953f6b77cf30e8f5d4a0a34fd3a74bL47). Things had changed when we had our initial jib-core public API, and what this class has been doing since then is just "jibConatinerBuilder.containerize(containerizer)". I think the name "JibRunner" [Tad also suggested](https://github.com/GoogleContainerTools/jib/pull/1659#discussion_r278749078) fits well?

Reply:
Sure, I was working under the assumption that using `Jib` doesn't add much actual descriptive value given that this is already in the `Jib` project namespace and so `JibRunner` can mean anything related to Jib?. But again, this is minor and I understand we can disagree.

Reply:
I get what you're saying. If we had a practice to prefix "Jib" to anything like "JibBuildConfiguration", "JibCredential", I agree that's meaningless as we are under the Jib namespace and we should avoid that. But since we don't execute such practice, I think "JibRunner" can carry the meaning of someone who runs Jib.

That said, maybe I should go with "JibBuildRunner". I had thought about "BuildRunner" but didn't like it because it is not descriptive enough either; it's not clear what kind of build it runs, given that there are too many things that are buildable. That is, it's unclear what the build result is. Here, I'd like to make it explicit that what we run is (a Jib build).